packages/components/src/components/FunctionChart/NumericFunctionChart.tsx

```diff
@@ -44,10 +44,20 @@ export const NumericFunctionChart: FC<Props> = ({
       context.clearRect(0, 0, width, height);
 
       const xScale = sqScaleToD3(plot.xScale);
-      xScale.domain(d3.extent(functionImage, (d) => d.x) as [number, number]);
+      xScale.domain(
+        d3.extent(
+          functionImage.filter((d) => isFinite(d.x)),
```

Comment:
Not very important, but: I can't come up with a scenario when this is necessary.

`functionImage[number].x` values may come from (1) settings or (2) plot object's xScale min/max or from (3) function parameter domain's min/max. In cases (2) and (3), min/max can be infinite: `Plot.numericFn(..., xScale: Scale.linear({ min: 1 / 0, ... })`.

But in that case `getFunctionImage` will extrapolate all points to be infinite, and the output will be broken anyway.

Reply:
Makes sense, I wasn't sure.